### DISCUSSION TITLE
Speed up Autoortho with pre-download Cache files

### DISCUSSION THREAD

**Discussion by GEC42P** [kubilus1/autoortho#295] [Ideas]
Wouldn't it make sense to implement a function to download everything for a given area like EU into the cache and only check whether a file is newer then the existing one ?
What I see is that Autoo uses less diskspace than the Ortho4XP so 100 GB Cache .... it doesn't matter

Manfred

**Comment by d41k4n** [comment]
No, that's not how AO works. AO doesn't care and doesn't know if a file is "newer". All it cares about is whether or not the DDS files contain data for a specific mipmap level at the time when X-Plane requests it. Mipmap levels correspond to downloaded textures at different zoom levels.

The main reason AO scenery for the same area uses less disk space than with Ortho4XP is because the DDS files are "sparsely allocated" i.e. they only contain the mipmap levels requested by X-Plane whereas with Ortho4XP all mipmap levels are available all the time, regardless if X-Plane requires them or not.
